package.json

```diff
@@ -22,7 +22,7 @@
     "express": "^4.14.0",
     "express-session": "^1.15.0",
     "lru-native": "^0.4.0",
-    "message-factory": "21-23/message-factory#v2.4.0",
+    "message-factory": "21-23/message-factory#v2.5.0",
```

Comment:
In yarn lock file the version is still 2.4.0. Use `yarn remove message-factory` + `yarn add 21-23/message-factory#v2.5.0`